mem.so/CMakeLists.txt

```diff
@@ -113,6 +113,16 @@ add_compile_options(-DBUILD_VERSION_STRING="${BUILD_VERSION_STRING}")
 set(ROCBLAS_LIB "rocblas")
 set(HIP_HCC_LIB "amdhip64")
 
+#ROCBLAS VERSION CHECK FLAGS TO CHECK REORG VERSION 2.44.0
+add_compile_options(-DRVS_ROCBLAS_VERSION_FLAT=${RVS_ROCBLAS_VERSION_FLAT})
+if(RVS_ROCBLAS_VERSION_FLAT LESS 2044000)
```

Comment:
As setting of ROCBLAS_MODULE_NM_PREFIX and message was moved to main CMakeList.txt, this if-else section shall be removed to avoid redundancy.

Reply:
Done